### DISCUSSION TITLE
End of The Road for Readium – The Global Impact on Digital Publishing - Is it true?

### DISCUSSION THREAD

**Discussion by asthagarg2428** [readium/mobile#24] [Q&A]
I was searching for the epub viewer sdk and found this white paper on kitaboo's site saying "**End of The Road for Readium – The Global Impact on Digital Publishing**".

"_To the dismay of many loyal users around the world, Google recently revealed that it is planning to disable the use of Readium soon. One of the major reasons is that Google’s recent changes to their security rules —such as no inline scripts — have ended up in partially crippling the application. Google had stated back in 2017 about the changes._"

Here is the complete pdf - https://kitaboo.com/wp-content/uploads/2020/05/End-of-The-Road-for-Readium-The-Global-Impact-on-Digital-Publishing.pdf

Is readium going to end? Is readium mobile sdk for Android about to end?
Would like to hear more on this from your side

**Comment by mickael-menu** [comment]
This only concerned the Chrome App / extension, not related to the Readium Mobile Toolkit.

> This will be the final release of the Readium Chrome application. Google has warned that the Chrome framework (on which the app is based) is deprecated and will be disabled sometime in 2018. In addition, Google’s changes to their security rules (e.g. no inline scripts) has already partially crippled the application. Therefore we have made the decision to cease supporting and releasing the app. 
> https://readium.org/development/releases/

As for reading in the browser, [Readium Web](https://github.com/readium/web) – which is up and coming – will address this.
